deployment.rb

```diff
@@ -187,6 +187,18 @@ def hourofcode_hostname
     canonical_hostname('hourofcode.com')
   end
 
+  def circle_run_identifier
+    ENV['CIRCLE_BUILD_NUM'] ? "CIRCLE-BUILD-#{ENV['CIRCLE_BUILD_NUM']}-#{ENV['CIRCLE_NODE_INDEX']}" : nil
```

Comment:
Done! Do we also want the `CIRCLE_NODE_INDEX` to be in the `S3_LOGS_PREFIX`? https://github.com/code-dot-org/code-dot-org/blob/bd7cae98ce060d1dc1f5c9f4dbd88f74c1de36c2/dashboard/test/ui/runner.rb#L37